public/app/dashboard/profileController.js

```diff
@@ -133,7 +133,7 @@
             'color': $scope.color.ACTIVITY_TIMELINE_RED
           },
           {
-            'name': ' Creations',
+            'name': 'Recreates',
```

Comment:
Per comment above, those spaces are a terrible hack to help EventDrops deal with repeated headings.